spec/mongoid/criteria_spec.rb

```diff
@@ -3026,13 +3030,14 @@ class D
       end
 
       context 'when plucking the entire field' do
+        with_config_values(:legacy_pluck_distinct, true, false) do |val|
+          let(:plucked) do
+            Dictionary.all.pluck(:description)
+          end
 
-        let(:plucked) do
-          Dictionary.all.pluck(:description)
-        end
-
-        it 'returns all translations' do
-          expect(plucked.first).to eq({'en' => 'english-text', 'de' => 'deutsch-text'})
+          it 'returns all translations' do
+            expect(plucked.first).to eq(val ? {"de"=>"deutsch-text", "en"=>"english-text"} : 'deutsch-text')
```

Comment:
In the case when 'deutsch-text' is returned, the description on line 3038 is wrong. This should be expanded to two tests (`it` blocks) each with the correct title.

Reply:
done